### PR TITLE
Pull request Milestone 2 - Tic Tac Toe

### PR DESCRIPTION
**Pull request by bruna-genz**

### CODE REVIEW

**Comment by bruna-genz** [comment]
Hi! 
I made a lot of changes in my mock game. I added some conditions to check if the move is available and if it is a winning or a draw move.
However, now there are only a few conditions that run on the terminal because the others depend on variables that I will create afterward.

**Comment by bruna-genz** [comment]
I am working by myself in Ruby, because I had problems with my coding partner. The support team authorized me to do so.

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,157 @@
...
+    puts "Ok, then. The board has a 3x3 grid. Each board space is represented by a number from 1 to 9, like that:
+    
+                             1 | 2 | 3 
+                            -----------
+                             4 | 5 | 6 
+                            -----------
+                             7 | 8 | 9 
+    
+Each player will be represented by a mark, X and O. Every round, the players will choose between an available position.
+The player who suceeds in placing three of their marks in a horizontal, vertical, or diagonal row wins the game.
```
Comment (Torres-ssf) [inline bin/main.rb]: You even wrote a tutorial :+1:  :1st_place_medal: 
Love it

**Pull request merged**